### ISSUE TITLE
Connectionstring issues caused by _makeConnectionString method

### ISSUE THREAD

**Issue by pgpetrov** [pavelvlasov/koa-generic-session-mongo#10]
Hi,

Inside dist/store.js the method _makeConnectionString does:

```
return 'mongodb://' + host + ':' + port + ')/' + db + '?ssl=' + ssl;
```
That right parenthesis messes up my connection string and I doubt that is something useful.
I think it should be removed.
(row 259)
Regards

**Comment by whilefor** [comment]
+1, make me confuse of ')/'

**Comment by dickeylth** [comment]
+1, makes my mongodb connection throwed error.

**Comment by pavelvlasov** [comment]
fixed with #11
